packages/typespec-azure-resource-manager/lib/common-types/types.tsp

```diff
@@ -217,7 +217,7 @@ model OperationStatusResult {
   @doc("Percent of the operation that is complete.")
   @minValue(0)
   @maxValue(100)
-  percentComplete?: int32;
+  percentComplete?: float64;
```

Comment:
In the original issue it was mentioning `float32` but we also reference `float64` here https://github.com/Azure/typespec-azure/blob/a72d7efeacb6e6b8d64b9c65853ecc42558d77d5/packages/typespec-azure-resource-manager/lib/responses.tsp#L124

So probably good those 2 are consistent, so do we need to change the other to float32 or this one is good.

Reply:
https://github.com/Azure/azure-resource-manager-rpc/blob/8231d7df51aec87e6ccb5bcc04478136758b0a4c/v1.0/async-api-reference.md?plain=1#L164

Reply:
so sound like this fix is good then

Reply:
Yes, double and float64 are synonymous